### DISCUSSION TITLE
Which the minimal rigths have to have the "user" used by the hikaxpro_hacs integration?

### DISCUSSION THREAD

**Discussion by enrico-stronati** [petrleocompel/hikaxpro_hacs#93] [Q&A] [answered]
Hi to all.

I am starting to test thi (great) AX Pro integration, I intend to use it to grab the events by means of sensors linked to the alarm panel, usually when this one is in "not armed state". This could be useful to raise events like "the kitchen window is opened since 5 minutes", then switch off the warm (this is an example).

So, I not use the integration to arm/disarm the alarm, I prefer to mantains control out of the HA.

For this I appreciate to know which minimal "user rights" I have to assign to the user used for the integration and what type user must be it (admin, operator, ....).

Regards.

**Comment by petrleocompel** [comment]
Currently I did not test with anything else than admin. 
You can test on your own. We have an proposal for configuring this integration as data collection (to disable "alarm panel"). 
Watch issue #90

**Reply by petrleocompel** [reply, marked as answer]
To make arming and disarming possible via API you have to be an Admin. You can try different role. But just check `Pre-Check` of current `README.md`.

Make sure your system does not have `Installer` or it is not the same address as your `admin` user.
